sys/include/event/periodic.h

```diff
@@ -36,20 +36,24 @@
 #ifndef EVENT_PERIODIC_H
 #define EVENT_PERIODIC_H
 
+#include <assert.h>
 #include "event.h"
 #include "ztimer/periodic.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#define EVENT_PERIODIC_FOREVER      ZTIMER_PERIODIC_KEEP_GOING
```

Comment:
```suggestion
#define EVENT_PERIODIC_FOREVER      0
```

It's not immediately obvious (or necessary) that `ZTIMER_PERIODIC_KEEP_GOING` is 0

Reply:
Sure, Ok to squash afterwards?

Reply:
Sure, please squash